packages/table/src/quadrants/_quadrants.scss

```diff
@@ -51,9 +56,22 @@ $table-quadrant-scroll-container-overflow: 20px;
 }
 
 .bp-table-quadrant-main {
-  right: 0;
-  bottom: 0;
+  // use `relative` to ensure the table can size itself according to the MAIN quadrant's contents alone.
+  // other quadrants should remain `absolute` to ensure they're taken out of the document flow.
```

Comment:
👍 this is usually my go-to solution for cutting my components out of the layout flow